config/webpack.server.js

```diff
@@ -6,19 +6,48 @@ const merge = require('webpack-merge');
 const defaults = require('./webpack.defaults.js');
 const packageConfig = require('./packageConfig');
 
-module.exports = merge.smart({
-  module: {
-    rules: [{
-      test: /\.css$/,
-      use: [path.join(__dirname, '../lib/exportLocalsLoader.js')],
-    }, {
-      test: /\.scss$/,
-      use: [path.join(__dirname, '../lib/exportLocalsLoader.js')],
-    }],
-  },
-}, defaults, {
+const config = merge.smart(defaults, {
   target: 'node',
   mode: 'production',
+  module: {
+    rules: [
+      {
+        test: /\.css$/,
+        include: path.resolve('src'),
+        use: [
+          {
+            loader: 'css-loader',
+            options: {
+              exportOnlyLocals: true,
+            },
+          },
+        ],
+      },
+      {
+        test: /\.css$/,
```

Comment:
can these two css tests be merged to one if we pass an array of paths to include?

Reply:
no, we need different config for the `node_modules` paths. the `src` paths use our old postcss pipeline as well as `modules`, so classnames in external packages were transformed to something like `.___node_modules____foo-bar-baz`